client/public/index.html

```diff
@@ -9,7 +9,7 @@
       homescreen on Android. See https://developers.google.com/web/fundamentals/engage-and-retain/web-app-manifest/
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link rel="shortcut icon" href="%PUBLIC_URL%/logo.png">
```

Comment:
We're still going to need a `.ico` file, that's what most browsers will use for the icon. We can use something like [this](https://realfavicongenerator.net/) to make one.